src/backends/torch/torchlib.cc

```diff
@@ -231,7 +231,8 @@ namespace dd
       }
     else if (!_template.empty())
       {
-        throw MLLibBadParamException("template");
+        _module.create_native_template<TInputConnectorStrategy>(
+            _template, lib_ad, this->_inputc, this->_mlmodel, _device);
```

Comment:
No error is thrown if the template is not valid. Check if template is valid with `NativeFactory::valid_template_def(_template))`?

Reply:
The call is made earlier https://github.com/jolibrain/deepdetect/blob/feat_ml_vit_torch_native/src/backends/torch/torchlib.cc#L190
I left it there actually instead of moving it.

Reply:
Ok I see
The problem is, the template is checked with the presence of traced nets or proto file all at once in a single condition. 
So if the user specifies a wrong template name with a traced net (e.g. "bret" instead of "bert" 
🙂), dede will try to create a native template and fail silently

Reply:
You are right, will fix.

Reply:
done.